workflow/rules/utils.smk

```diff
@@ -41,6 +41,19 @@ rule bcf_index:
         "bcftools index {input} 2> {log}"
 
 
+rule bcf_sort:
+    input:
+        "{prefix}.bcf",
+    output:
+        "{prefix}_sorted.bcf",
```

Comment:
Suggestion: Should this maybe be a `{prefix}.sorted.bcf` so that we stay consistent with the other nomenclature? (e.g. `../{sample}.{varrange}.bcf`)